AWSIoT/Internal/AWSIoTMQTTClient.m

```diff
@@ -639,6 +672,12 @@ - (void)openStreams:(id)sender
     [defaultRunLoopTimer invalidate];
     
     if (!self.runLoopShouldContinue ) {
+        
+        if (self.connectionAgeTimer != nil ) {
```

Comment:
@cbommas, your earlier suggestion to simplify this was correct as this method will be called already on the correct thread.

Reply:
👍